### PR TITLE
Remove discovery junk

### PR DESCRIPTION
**Pull request by joesoftware**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,41 +1,9 @@
...
-```
-node weather-rest
-```
-
-then browse to [http://localhost:3000/hca/explorer](http://localhost:3000/hca/explorer)
-
-## Example 2 - simple Java SOAP server access (semi-works)
-
-### To start a simple soap server, run:
-
```
Comment (robbiethegeek) [inline README.md]: we should add the npm run watch for local development.

Comment (robbiethegeek) [inline README.md]: Also is there a design doc or other documentation.

**Comment by robbiethegeek** [comment]
If you can add minor tweaks to the README.md I think this is good and will clear up some of the confusion.

**Comment by akainic** [comment]
@joesoftware LGTM! Please squash commits and merge.

**Comment by robbiethegeek** [comment]
Agree with @akainic squash and merge 👍 LGTM

**Pull request merged**